madmom/ml/nn/layers.py

```diff
@@ -44,16 +51,24 @@ def activate(self, data):
         """
         raise NotImplementedError('must be implemented by subclass.')
 
+    @staticmethod
```

Comment:
Not sure this should be a static method - sure, it does not change the object state in the `Layer` class, but its functionality is **intended** to change the object state.